### ISSUE TITLE
Make destroying the tendril kill all ashwalkers

### ISSUE THREAD

**Issue by matskuman5** [yogstation13/yogstation#1796]
Or at least give them a major slowdown and weakness

This would make it so protecting the tendril is more important, instead of murderboning the escape shuttle

This would ALSO fix the moment when tendril is destroyed and ashwalkers start murderboning because "lel no tendril to drag bodies to"

**Comment by Jc1469** [comment]
Balance police say fuck off. This basically forces the ashwalkers to not leave, which makes the role reduntant. :-1:

**Comment by Time-Green** [comment]
Yeah, imagine ashwalkers being forced to not go to the station and kill everyone

**Comment by puremanbird** [comment]
not a terrible idea.

**Comment by YogsMcree** [comment]
It doesn't force them to do anything. Most players don't know WHERE the damn thing is. Those who do have a chance to be outrobusted by the ashwalkers and if you are murderboning your FIRSTBORN priority should've been the miners who can get op loot.

**Comment by ClaudeWicked** [comment]
This is a dumb idea, as far as Im concerned. Barring lag spikes, miners have a smorgasbord of loot far above and beyond that which can be found on the station. Even an unupgraded KA makes dealing with ashwalkers trivial, and having a 'Delete all the enemies" button seems dumb, both mechanically, and in the little roleplay this server has.  McCree's response, for example, is wrong- Ashwalkers will, 90% of the time, start developing blood trails which lead to their base. This might make sense if Ashwalkers were as strong as they used to be, and if the tendril was a decent bit more resilient, and alerted the ashwalkers when it was being attacked. But right now, killing the tendril is a gamble-- it stops further ashwalkers from spawning, and gives a very useful anomaly core, but it also leaves the remaining ashwalkers with nothing on their mind but vengeance.

**Comment by Time-Green** [comment]
I think the best idea would be to give ashwalkers a permanent nerf if this were too happen

**Comment by ShadowDeath6** [comment]
tbh not even a bad idea, it should just kill all the ashwalkers off the mining z

**Comment by Time-Green** [comment]
Shadow, you're a genious

**Comment by ClaudeWicked** [comment]
I actually don't mind that, Shadow, but it seems a bit gamey.

**Comment by Dagdammit** [comment]
I was gonna suggest the same thing as shadow- maybe they start constantly taking cold damage while on other z levels? Something that gives a full health ash walker like a minute & change to make it back to lavaland.

What if smashing the tendril does that and prevents the spawning of new chieftan eggs, but the tendril stump remains & will still gib corpses and spawn new regular eggs? Possibly at a reduced rate, i.e. requires twice as many bodies. 

Miners could still wall it off or maybe attack tendril stump more to smash it entirely? I'd just appreciate having an option to defang the ash walkers without having to commit outright genocide.

**Comment by puremanbird** [comment]
just lower their speed by a noticeable amount while their on another z level. Meme it by saying its some psychic link to the holy tendril. Now they're balanced and less cancerous.

**Comment by Dagdammit** [comment]
Reducing them to say, golem-tier speeds is an interesting thought, I like
it.

**Comment by ClaudeWicked** [comment]
Honestly, a flat speed nerf to golem levels when outside lavaland sounds reasonable, both balance wise and and roleplay.

**Comment by GrayRachnid** [comment]
They're REPTILES reptiles MOVE FAST

**Comment by Dagdammit** [comment]
ESPECIALLY in places that are COLDER than their NATIVE ENVIRONMENTS

**Comment by puremanbird** [comment]
they're CANCER cancer SPREADS FAST. seriously though, i cant tell if you're being serious or not gray.

**Comment by ClaudeWicked** [comment]
I think they should retain their speed on lavaland, but making them slower off it makes them manageable without going "Hurr you're dead."
